cmd/kubelet/app/server.go

```diff
@@ -926,7 +926,11 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 	}
 
 	// If systemd is used, notify it that we have started
-	go daemon.SdNotify(false, "READY=1")
+	go func() {
+		if _, err := daemon.SdNotify(false, daemon.SdNotifyReady); err != nil {
+			klog.ErrorS(err, "Failed to send systemd daemon successful start message")
```

Comment:
will we see this error all the time when running outside of systemd? Or it will only fail when running on systemd and has issues with notifying?

Reply:
<img width="716" alt="Screenshot 2024-10-21 at 08 55 35" src="https://github.com/user-attachments/assets/85abffc0-0c47-4ba7-8a58-d2ee9d3e4b98">
according to `daemon.SdNotify`, if the `NOTIFY_SOCKET` is not set, the func will return nil. since the env is only set when running on systemd(Type=notify), the func will only work when running on systemd.